Automattic/sensei
# Implement Course Theme  Quiz_Graded block.

Fixes #4450 

### Changes proposed in this Pull Request

* Implements a Quiz Graded block for the Course Theme.
* The design at #4450 shows "Complete lesson" button, but the lesson is already marked as complete after you pass a quiz or if you submit a quiz that does not require passing. So not sure when that button supposed to show up. @renatho @sruj09 WDYT?

### Testing instructions


* Enable the feature flag: `add_filter( 'sensei_feature_flag_course_theme', '__return_true' );`
* Create a lesson with a quiz in it and turn on auto grading. (Or do manual grading, doesn't matter.)
* Add this lesson to a course that has Course Theme enabled.
* Visit lesson page, click View Quiz and complete the quiz.
* Confirm that there is a "Quiz Graded" block rendered at the top. (see video) 


### Screenshot / Video


https://user-images.githubusercontent.com/2578542/146360850-dbc573a9-d972-4983-9d62-13647e7e6408.mp4
